iree/samples/static_library/CMakeLists.txt

```diff
@@ -0,0 +1,54 @@
+# Copyright 2021 The IREE Authors
+#
+# Licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+# if(${IREE_BUILD_STATIC_LIBRARY_SAMPLES} AND NOT ${IREE_BUILD_SAMPLES})
+#   return()
+# endif()
+
+if(NOT ${IREE_TARGET_BACKEND_DYLIB-LLVM-AOT}
+  OR NOT ${IREE_HAL_DRIVER_DYLIB}
+  OR NOT ${IREE_BUILD_COMPILER}
+  OR NOT ${IREE_BUILD_SAMPLES})
+  return()
+endif()
+
+iree_bytecode_module(
+  NAME
+    simple_mul
+  SRC
+    "simple_mul.mlir"
+  C_IDENTIFIER
+    "iree_samples_static_library_simple_mul"
+  FLAGS
+    "-iree-input-type=mhlo"
```

Comment:
You could switch the file to using `std` (Standard) dialect ops instead of MHLO then drop this flag. We want to encourage people to either use a frontend importer (`iree-import-tf`) that happens to use mhlo internally _or_ to use the dialects that IREE advertises support for (Standard, Linalg, Math, etc.)

https://mlir.llvm.org/docs/Dialects/Standard/#stdmulf-mlirmulfop